projectile.el

```diff
@@ -1222,27 +1222,22 @@ Files are returned as relative paths to DIRECTORY."
 
 The function calls itself recursively until all sub-directories
 have been indexed.  The PROGRESS-REPORTER is updated while the
-function is executing.  The list of IGNORED-FILES and
```

Comment:
I'm guessing something got merged incorrectly by looking at the diff.

Reply:
Should be fixed now, thanks for the spot